runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -150,6 +149,10 @@ type configProperties struct {
 	Provision bool `mapstructure:"provision"`
 	// DSN is the connection string. Either DSN can be passed or the individual properties below can be set.
 	DSN string `mapstructure:"dsn"`
+	// ReadDSN is the connection string for read operations. Takes precedence over DSN for reads.
+	ReadDSN string `mapstructure:"read_dsn"`
+	// WriteDSN is the connection string for write operations. Takes precedence over DSN for writes.
+	WriteDSN string `mapstructure:"write_dsn"`
```

Comment:
Seems like there's still three DSNs here? Shouldn't it just be `DSN` and `WriteDSN` now?